src/python/tests/signature_tests.py

```diff
@@ -4,58 +4,59 @@
 import pandas as pd
 from demoparser2 import DemoParser
 
+demo_path = "../parser/test_demo.dem"
 
 class SignatureTest(TestCase):
     def test_parse_header_signature(self):
```

Comment:
I think a straight up string path in a file is always relative to the location that you are calling the file from, not the path of the file.

Reply:
Shouldn't it be correct then? we are in src/python and want to refer to parser/test_demo.dem?

Reply:
Ah, yeah. I didnt read it fully.